make-file-list.sh

```diff
@@ -156,6 +156,7 @@ grep -v -e '\.[acho]$' -e '\.l[ao]$' -e '/aclocal/' \
 	-e '^/usr/share/perl5/core_perl/TAP/' \
 	-e '^/usr/share/vim/vim74/lang/' \
 	-e '^/etc/profile.d/git-sdk.sh$' |
+grep -v -P '^/usr/share/awk(?!/inplace\.awk$)' |
```

Comment:
How about dropping this?

```sh
$ du -hs /usr/share/awk/
54K     /usr/share/awk/
```

That's not really anything to worry over, it will be compressed well by LZMA.

Reply:
Sounds good to me. :) I'll update the commit to address both items this evening, and force-push the PR branch.

Reply:
👍